### PR TITLE
R1 Final Review

### PR DESCRIPTION
**Pull request by amsmith-pro**
Adds running from a configuration file and eel file

### CODE REVIEW

**Comment by owlot** [comment]
@cla-bot check

**Comment by owlot** [comment]
## Documents
- [x] README
  - [x] Badges
  - [x] Accurate installation instructions
  - [x] Current links to other documents
  - [x] Arrow DAO links

## Code (not applicable to examples/ folder)
- [ ] No unhandled unwrap()s
- [ ] Accurate (!) code comments on all functions
- [ ] No `# Arguments` and `# Returns` in function comments
- [ ] Function arguments checked where possible
- [ ] Sufficient logger (info, warn, and debug) messages
- [ ] Sensible Types (unsigned vs. signed)

## Cargo.toml(s)
- [x] Package versions correct
- [x] No minor versions specified
- [x] Links to crates.io package, not git branch

File: README.md [hunk 1]
```diff
@@ -1,2 +1,93 @@
 # tool-simulation
```
Comment (owlot) [inline README.md]: Missing header, badges and Arrow DAO links

File: sim/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name    = "tool-simulation"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```
Comment (owlot) [inline sim/Cargo.toml:8]: patch versions used, should be changed to major + minor only

File: tools/validate_config/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[package]
+edition = "2021"
+name    = "validate_config"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```
Comment (owlot) [inline tools/validate_config/Cargo.toml:8]: patch versions used, should be changed to major + minor only

File: tools/validate_eel/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[package]
+edition = "2021"
+name    = "validate_eel"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```
Comment (owlot) [inline tools/validate_eel/Cargo.toml:8]: patch versions used, should be changed to major + minor only

File: types/Cargo.toml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name    = "sim_types"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```
Comment (owlot) [inline types/Cargo.toml:8]: patch versions used, should be changed to major + minor only

**Review by owlot — CHANGES_REQUESTED** [review]
Most of the `Code` checks can't be ticked off. Since this is a tool, we can skip that for R1, but it would be good to make tickets for these to take care of in a next release.

File: sim/Cargo.toml (hunk 2 above)
Comment (amsmith-pro) [reply sim/Cargo.toml:8]: 748d68a6c35bab95af22594b3a08aa94f1788c59

File: tools/validate_config/Cargo.toml (hunk 3 above)
Comment (amsmith-pro) [reply tools/validate_config/Cargo.toml:8]: 564ebfb24e324f1c98f47b94587a72cf0d550abf

File: tools/validate_eel/Cargo.toml (hunk 4 above)
Comment (amsmith-pro) [reply tools/validate_eel/Cargo.toml:8]: 564ebfb24e324f1c98f47b94587a72cf0d550abf

File: types/Cargo.toml (hunk 5 above)
Comment (amsmith-pro) [reply types/Cargo.toml:8]: 564ebfb24e324f1c98f47b94587a72cf0d550abf

**Comment by amsmith-pro** [comment]
@owlot Made the fixes! Sorry, should have kept this in draft 

depends on svc-cargo main branch, will rerun CI when that's merged in

File: README.md (hunk 1 above)
Comment (amsmith-pro) [reply README.md]: fixed

**Review by owlot — APPROVED** [review]

**Pull request merged**